<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileConfigTab</class>
 <widget class="QWidget" name="ProfileConfigTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Load profile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="profileCombo">
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="loadProfileButton">
     <property name="text">
      <string>&amp;Load profile</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="saveProfileButton">
     <property name="text">
      <string>&amp;Save as...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frequency</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="FrequencySpinBox" name="frequencySpinBox"/>
   </item>
   <item row="2" column="2">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_20">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="iqBalanceCheck">
     <property name="text">
      <string>I/Q Balance</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="removeDCCheck">
     <property name="text">
      <string>DC remove</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>   LNB frequency</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="FrequencySpinBox" name="lnbSpinBox"/>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="widget_6" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_21">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_46">
     <property name="text">
      <string>Freq. correction</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QDoubleSpinBox" name="ppmSpinBox">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> ppm</string>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Sample rate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QStackedWidget" name="sampRateStack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="sampRateSdrPage">
      <layout class="QGridLayout" name="gridLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QComboBox" name="sampleRateCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sampRateFilePage">
      <layout class="QGridLayout" name="gridLayout_12">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="FrequencySpinBox" name="sampleRateSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_75">
     <property name="text">
      <string>True rate:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="trueRateLabel">
     <property name="text">
      <string>N/A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Decimation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="decimationSpin">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> samples</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_49">
     <property name="text">
      <string>Source time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QDateTimeEdit" name="sourceTimeEdit">
     <property name="minimumSize">
      <size>
       <width>173</width>
       <height>0</height>
      </size>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="displayFormat">
      <string> dd/MM/yyyy HH:mm:ss.zzz</string>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Analyzer type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="analyzerTypeCombo">
     <item>
      <property name="text">
       <string>Local</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Remote</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QCheckBox" name="sourceTimeIsUTCCheck">
     <property name="text">
      <string>Time is &amp;UTC</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="QStackedWidget" name="analyzerParamsStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="localPage">
      <layout class="QGridLayout" name="gridLayout_14">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QRadioButton" name="sdrRadio">
         <property name="text">
          <string>S&amp;DR device</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="sdrFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Device</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="channelSpin"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Channel #</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Bandwidth</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="deviceCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="modelColumn">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Antenna</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="antennaCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="FrequencySpinBox" name="bandwidthSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="fileRadio">
         <property name="text">
          <string>&amp;File source</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QFrame" name="fileFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>File</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="pathEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="formatCombo">
            <item>
             <property name="text">
              <string>Autodetect</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Raw complex 32-bit float (CF32)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Raw complex 8-bit unsigned (CU8)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Raw complex 16-bit signed (CS16)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>WAV file</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="browseButton">
            <property name="text">
             <string>&amp;Browse...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="loopCheck">
            <property name="text">
             <string>&amp;Loop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_16">
       <item row="0" column="0" colspan="8">
        <widget class="QRadioButton" name="useNetworkProfileRadio">
         <property name="text">
          <string>Connect to LAN profile</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLineEdit" name="hostEdit">
         <property name="text">
          <string>localhost</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_44">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="4">
        <widget class="QLineEdit" name="userEdit">
         <property name="text">
          <string>anonymous</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_47">
         <property name="text">
          <string>Profile name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QSpinBox" name="portEdit">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>28001</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>&amp;Refresh</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_43">
         <property name="text">
          <string>User</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="2" colspan="4">
        <widget class="QLineEdit" name="passEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>(no password)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="4">
        <widget class="QComboBox" name="remoteDeviceCombo"/>
       </item>
       <item row="2" column="0" colspan="8">
        <widget class="QRadioButton" name="useHostPortRadio">
         <property name="text">
          <string>Connect manually to host</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Port</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="6" rowspan="3" colspan="2">
        <widget class="QLabel" name="label_45">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Support for remote analyzers is provided by an experimental application layer protocol that is still under active development and lacks basic security features like proper authentication or transport encryption. Use at your own risk.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrequencySpinBox</class>
   <extends>QWidget</extends>
   <header>FrequencySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
